java/common/rules/java_runtime.bzl

```diff
@@ -152,6 +152,8 @@ def _java_runtime_rule_impl(ctx):
         platform_common.TemplateVariableInfo({
             "JAVA": java_binary_exec_path,
             "JAVABASE": java_home,
+            "JAVA_ROOTPATH": paths.join(java_home_runfiles_path, "bin/java"),
```

Comment:
This won't work on Windows. Could you use `java_binary_runfiles_path` instead? If that somehow isn't the right path, you could also reuse the basename of the execpath.

Reply:
Done, `java_binary_runfiles_path` does actually work, the earlier issue I had must have been due to the `JAVA_RUNFILES` name conflict.